src/differencer/diffProcessor.ts

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
can we rename to just difftool from differencer?

Reply:
sure